core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -617,4 +653,20 @@ private static String bossThreadName(ServerPort port) {
                                          .collect(Collectors.joining("+"));
         return "armeria-boss-" + protocolNames + '-' + localHostName + ':' + localAddr.getPort();
     }
+
+    private static boolean isLocalPort(ServerPort serverPort, @Nullable SessionProtocol protocol) {
+        if (!serverPort.localAddress().getAddress().isAnyLocalAddress()) {
+            return false;
+        }
+
+        if (!serverPort.localAddress().getAddress().isLoopbackAddress()) {
```

Comment:
You could extract `serverPort.localAddress().getAddress()` into a variable to reduce code duplication.